api/api_errors.go

```diff
@@ -98,7 +98,7 @@ func (ErrActorNotFound) Error() string { return "actor not found" }
 
 type errF3Disabled struct{}
 
-func (errF3Disabled) Error() string { return "f3 is disabled" }
+func (errF3Disabled) Error() string { return "F3 is disabled" }
```

Comment:
Capitalised error message is against the common Go practices that I am aware of.

I don't know how consistently this is followed in Lotus repo. But I can see even in this PR, error message casing is inconsistent.

Reply:
As per comment at the top of the PR, this is to align with go-f3 (see https://github.com/filecoin-project/go-f3/blob/1b17c6c40971e9ff408afa8b8cd7fb90f79e587c/f3.go#L119-L127); I wouldn't mind keeping these lower-case but it's harder to ix in go-f3 than just make it aligned here, and capitalising acronyms is OK and this is kind of an acronym so we're in one of those cases where I reckon it's OK to go either way

Reply:
And re lotus conventions - we pretty consistently stick to the lower-case error convention; there's only a few stray cases left in the repo

Reply:
FWIW i have captured https://github.com/filecoin-project/go-f3/issues/795

Reply:
ok, if you're making as a bug over there I'll lower-case it here and then we can have them aligned when we eventually get an update 👍